src/ui/ui_manager.py

```diff
@@ -51,10 +51,11 @@ def use_wp(self, act: int, idx: int):
         wait(0.4, 0.5)
         mouse.click(button="left")
         # wait till loading screen is over
-        if self.wait_for_loading_screen():
+        if self.wait_for_loading_screen(3):
```

Comment:
This was causing a game timeout by the loop just sitting and waiting for the loading screen. But if you exited the wp menu after opening it (like I'm seeing happen sometimes) it will just sit. This now exits the game